src/components/MapView/Legends/index.tsx

```diff
@@ -228,7 +228,11 @@ function LegendItem({
                 legend.map(({ value, color }: any) => (
                   <ColorIndicator
                     key={value}
-                    value={value as string}
+                    value={
+                      typeof value === 'number'
+                        ? Math.round(value).toLocaleString('en-US')
```

Comment:
is `number.toString()` ok here? Or is `toLocaleString` better for this case?

Reply:
toLocateString includes the thousand separator. For instance 2000 as number will give you '2,000'